connectivity/check/check.go

```diff
@@ -1395,7 +1395,7 @@ func (k *K8sConnectivityCheck) deploy(ctx context.Context) error {
 			Name:   echoSameNodeDeploymentName,
 			Kind:   kindEchoName,
 			Port:   8080,
-			Image:  "quay.io/cilium/json-mock:1.2",
+			Image:  "quay.io/cilium/json-mock:v1.3.0@sha256:2729064827fa9dbfface8d3df424feb6c792a0ba07117b844349635c93c06d2b",
```

Comment:
Might make sense to define this in package `defaults` so it doesn't have to be repeated.